### PR TITLE
Improvement to the tests suggested in pull request 631

### PR DESCRIPTION
**Pull request by Gregory-F**
I tried to improive the tests suggested in the pull request #631 for is_full_window.

Now a list of windows is tested (represented by strings) and if no windows are passed as arguments for set_full_screen() is tested.

PS:
i did a first pull request with the same mistakes, i did not checked befor. I am sorry for this wast of your time. I hope you will forgive my beginer mistake.

## PR Checklist:
- [x] All new features have been tested #all tests passed
- [x] All new features have been documented #no new features
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: src/core/tests/test_app.py [hunk 1]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
...
+
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+
+        self.app.set_full_screen(["window1", "window2", "window3"])
+        self.assertTrue(self.app.is_full_screen)
+
+        self.app.set_full_screen()
+        self.assertFalse(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 2]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
         self.assertEqual(self.app.main_window.app, self.app)
         self.assertActionPerformed(self.app.main_window, 'show')
 
+    def test_is_full_screen(self):
+        self.assertFalse(self.app.is_full_screen)
+        
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 3]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
...
 
+    def test_is_full_screen(self):
+        self.assertFalse(self.app.is_full_screen)
+        
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+        
+        self.app.set_full_screen(["window1", "window2", "window3"])
+        self.assertTrue(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 4]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
...
+        
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+        
+        self.app.set_full_screen(["window1", "window2", "window3"])
+        self.assertTrue(self.app.is_full_screen)
+        
+        self.app.set_full_screen()
+        self.assertFalse(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 5]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
         self.assertEqual(self.app.main_window.app, self.app)
         self.assertActionPerformed(self.app.main_window, 'show')
 
+    def test_is_full_screen(self):
+        self.assertFalse(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 2 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 3 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 6]
```diff
@@ -60,6 +60,18 @@ def test_app_startup(self):
...
+        
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+        
+        self.app.set_full_screen(["window1", "window2", "window3"])
+        self.assertTrue(self.app.is_full_screen)
+        
+        self.app.set_full_screen()
+        self.assertFalse(self.app.is_full_screen)
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 5 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 2 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 3 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 4 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 5 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 2 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 3 above)
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py (hunk 1 above)
Comment (GrayR00t) [reply src/core/tests/test_app.py]: When you press enter to move to next line it created some tab spaces just delete it .

**Pull request merged**